src/entities/Proposal/jobs.ts

```diff
@@ -97,6 +93,35 @@ function updateCategoryBudget(proposal: ProposalWithOutcome, budgetForProposal:
   budgetForProposal.allocated += size
 }
 
+async function getProposalsWithOutcome(proposals: ProposalAttributes[], context: JobContext) {
+  const pendingProposalsWithOutcome = []
+
+  for (const proposal of proposals) {
+    const outcome = await calculateOutcome(proposal, context)
+    if (!outcome) {
+      continue
+    }
+
+    pendingProposalsWithOutcome.push({ ...proposal, ...outcome })
+  }
+
+  return pendingProposalsWithOutcome
+}
+
+async function getFinishedTenderProposals(pendingProposals: ProposalAttributes[]) {
```

Comment:
This is weird because it's called `getFinishedTenderProposals` but returns pending tender proposals.
I'd try to avoid using pending if it's going to be a proposal status.

How about getFinishableTenderProposals? Or something that indicates that they are proposals that need to be processed.